plugins/woocommerce/includes/class-wc-install.php

```diff
@@ -2284,13 +2284,13 @@ private static function set_paypal_standard_load_eligibility() {
 	 */
 	private static function get_refunds_return_policy_page_content() {
 		return <<<EOT
-<!-- wp:paragraph -->
-<p><b>This is a sample page.</b></p>
-<!-- /wp:paragraph -->
+<!-- wp:heading -->
+<h3 class="wp-block-heading">This is a sample page.</h3>
+<!-- /wp:heading -->
```

Comment:
```suggestion
<!-- wp:paragraph -->
<p><b>This is a sample page.</b></p>
<!-- /wp:paragraph -->
```

Even though this one is also bolded, I don't think it's intended to be a header, but rather just a note to the site owner, so I think this should stay as a paragraph.